nc_test4/unfilteredvv.cdl

```diff
@@ -0,0 +1,29 @@
+netcdf unfilteredvv {
```

Comment:
The convention in the nc_test4 directory is that CDL files used for tests start with "ref_", short for "reference", so that it's easy to see which CDL files come with the distro and which are generated during tests.